Update policies-support-lifecycle.md (#2747)
replicatedhq/replicated-docs: docs/vendor/policies-support-lifecycle.md

    <th width="30%">Product Phase</th>
    <th width="70%">Definition</th>
  </tr>
  <tr>
    <td>Alpha</td>

>     <td>A product, or a feature within a product, that is exploratory/experimental. It is typically not publicly documented, and access is limited to customers providing early feedback. While most alpha features will move forward, some may be deprecated based on assessment learnings. Products that are GA themself may have certain alpha features for a limited time as new capabilities are introduced to that product.</td>

Author: * Good to remove mention about docs. I think that is implied enough by limited access in this case
* My concern in removing the call call about about GA products can have alpha or beta features is that it feels like we have to explain this A LOT. A product like CMX or EC can be GA, but there will be features within that product that are alpha or beta. Not everything is automatically GA just because the parent product is. 
* No, there is not a predictable or standard timeline to communicate. It varies greatly. I more wanted to imply that it's not forever and that the feature's status will change one way or another.

>     <td>A product, or a feature within a product, that is exploratory/experimental. It is typically not publicly documented, and access is limited to customers providing early feedback. While most alpha features will move forward, some may be deprecated based on assessment learnings. Products that are GA themself may have certain alpha features for a limited time as new capabilities are introduced to that product.</td>

Reviewer: I kind of liked having the docs detail in there. Maybe something like:
> Typically, access to alpha features and their documentation is limited to customers providing early feedback.

>     <td>A product, or a feature within a product, that is exploratory/experimental. It is typically not publicly documented, and access is limited to customers providing early feedback. While most alpha features will move forward, some may be deprecated based on assessment learnings. Products that are GA themself may have certain alpha features for a limited time as new capabilities are introduced to that product.</td>

Reviewer 2: Oops, ignore that ^ I think I posted this comment just after you pinged us to say it's ready for re-review :)

  </tr>
  <tr>
    <td>Beta</td>

>     <td><p>A product, or a feature within a product, that has not yet met Replicated’s definition of GA. Products that are GA themself may have certain Beta features for a limited time as new capabilities are introduced to that product.</p><p>A product/feature may be labeled Beta for one or more of the following reasons: remaining gaps in intended functionality, outstanding needs around testing, documentation, or sales enablement, or still in-progress customer value validation efforts. Documentation for these features/products are published as part of the official documentation with Beta labeling, and many are already production ready depending on their development lifecycle phase.</p><p>Please contact Replicated if you have specific questions about why a product/feature is currently labeled Beta.</p></td>

Author: Similar comment above on 'a ga product may have beta features' edit

My initial hesitation on that "typically production ready" edit is that this is more true of BETA work in the last 2-3 years but not very true of some of the long-standing KOTS beta features. Those will age out though. I think your vision will probably resonate better with the vendor reading it and is technically true by the use of the word "typically"

>     <td><p>A product, or a feature within a product, that has not yet met Replicated’s definition of GA. Products that are GA themself may have certain Beta features for a limited time as new capabilities are introduced to that product.</p><p>A product/feature may be labeled Beta for one or more of the following reasons: remaining gaps in intended functionality, outstanding needs around testing, documentation, or sales enablement, or still in-progress customer value validation efforts. Documentation for these features/products are published as part of the official documentation with Beta labeling, and many are already production ready depending on their development lifecycle phase.</p><p>Please contact Replicated if you have specific questions about why a product/feature is currently labeled Beta.</p></td>

Reviewer: The "typically production ready" language addresses my quality comment. Maybe something at the end of beta (and alpha?) like:

> Software released in the Beta phase follows the same build and test processes as software released in the GA phase and is generally of production quality.

>     <td><p>A product, or a feature within a product, that has not yet met Replicated’s definition of GA. Products that are GA themself may have certain Beta features for a limited time as new capabilities are introduced to that product.</p><p>A product/feature may be labeled Beta for one or more of the following reasons: remaining gaps in intended functionality, outstanding needs around testing, documentation, or sales enablement, or still in-progress customer value validation efforts. Documentation for these features/products are published as part of the official documentation with Beta labeling, and many are already production ready depending on their development lifecycle phase.</p><p>Please contact Replicated if you have specific questions about why a product/feature is currently labeled Beta.</p></td>

Author: Let me see how I can try to work that last suggestion in without it getting repetitive. I think it's generally accurate for Beta features/product but not Alpha, which can include purposely not yet production ready experimental feature code.

  </tr>
  <tr>
    <td>“GA” - General Availability</td>

>     <td>A product, or a feature within a product, that has been validated as both production ready and value additive by a percentage of Replicated customers. For high-level products, the GA phase is typically when they become available for purchase from Replicated.</td>

Reviewer 2: ```suggestion
    <td>A product or feature that has been validated as both production-ready and value-additive by a percentage of Replicated customers. Products in the GA phase are typically those that are available for purchase from Replicated.</td>
```
Not sure if my edit on that last sentence gets to the heart of what you wanted to say or not.

I could also see a brief definition of what we mean by "production-ready and value-additive" potentially being helpful. For example, something like: "production-ready refers to features and products that meet the security, reliability, and performance needs of users"
